config/jenkins_weekly.ci.jenkins.io.yaml

```
@@ -21,6 +21,9 @@ controller:
   image: jenkinsciinfra/jenkins-weekly
   tag: 0.116.1-2.407
   pullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: linux
+    kubernetes.azure.com/agentpool: arm64small
```

```suggestion
    kubernetes.azure.com/agentpool: x86medium
```

We haven't evaluated the JVM behavior with Jenkins controller under ARM. Too risky

---

It's a demo instance, wouldn't it be a perfect candidate to test it as no production impact if something goes wrong?

Could be in another PR

---

It's way way way out of our scopes

---

definitely worth putting it on ARM doesn't have to be this PR